packages/manager/modules/pci/src/projects/project/storages/databases/database/namespaces/add/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
**issue(blocking):** Looks like we introduced some "static" content (English content) into a translation file (French).

For instance we have (all are not listed):
- "pci_databases_namespaces_add_type_aggregated": "Aggregated",
- "pci_databases_namespaces_add_type_unaggregated": "Unaggregated",
- "pci_databases_namespaces_add_writes_commitlog_true": "True",
- "pci_databases_namespaces_add_writes_commitlog_false": "False",
- "pci_databases_namespaces_add_snaphot_enabled_label": "snapshotEnabled"
-  "pci_databases_namespaces_add_snaphot_enabled_default": "Default",
- "pci_databases_namespaces_add_snaphot_enabled_true": "True",
- "pci_databases_namespaces_add_snaphot_enabled_false": "False",
- "pci_databases_namespaces_add_blocksize_duration_label": "retention.blockSizeDuration",
- ...

If the content is not meant to be translated, then it's not a translation. It's a static content/or constants.

Can we extract them from this translation file?

Reply:
Done here: https://github.com/ovh/manager/pull/6414/commits/9a77745e269c11e837cbecc64d1fb45817ca92a2